public/static/docs/command-reference/config.md

```diff
@@ -119,7 +119,7 @@ for more details.) This section contains the following options:
   this config option on forces you to run `dvc unprotect` before updating a
   file, providing an additional layer of security to your data.
 
-  It's highly recommended to enable this mod when `cache.type` is set to
+  It's highly recommendable to enable this option when `cache.type` is set to
```

Comment:
recommended vs recommendable - 900K vs 12K in google.

Reply:
"It's recommended" by who? 🙂 
I think we should either say "It's recommendable" or "We recommend".

Reply:
I thinks it's just a common phrase more or less - https://www.google.com/search?q=%22It%27s+highly+recommended+to+use%22&oq=%22It%27s+highly+recommended+to+use%22&aqs=chrome..69i57.4247j0j7&sourceid=chrome&ie=UTF-8 . I does feel natural even w/o specifying "by who" part.

On the other hand - `It's recommendable` is extremely rare.

I'm not agains `We recommend` if we can get a prove that `It's recommended` is wrong.

Reply:
900k results is not rare :P but fine, no reason to change something if it's not wrong. It's just not literally correct (again, recommended by who?) but if it's a common phrase then no problem. Will revert.

Reply:
I mean 12k

Reply:
OK I reviewed the usage of this term throughout all docs now. Feel fee to review again. Thanks